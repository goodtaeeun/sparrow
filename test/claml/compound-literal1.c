int *p = (int[]){2, 4};
int q[2] = {2, 4};
int g = (int)1.2;
