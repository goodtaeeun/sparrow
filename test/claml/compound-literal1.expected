/* Generated by CIL v. 1.7.4 */
/* print_CIL_Input is false */

#line 1 "compound-literal1.c"
int tmp[2] = {2, 4};
#line 1 "compound-literal1.c"
int *p = (int *)tmp;
#line 2 "compound-literal1.c"
int q[2] = {2, 4};
#line 3 "compound-literal1.c"
int g = (int)1.2;
